src/setupTests.js

```diff
@@ -3,3 +3,8 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom/extend-expect';
+
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
```

Comment:
Thank you for getting this set up for us so we have the set up out of the way for testing.